### PR TITLE
Implement Fixed Part of Landing Page

### PR DESCRIPTION
**Pull request by Lhyejin**
issue #29 
- Landing Page Main Header
- Sub Explanation
- View Button

### CODE REVIEW

File: web/src/App.js [hunk 1]
```diff
@@ -9,6 +8,8 @@ import {
     Link
 } from "react-router-dom"
 
+
```
Comment (shberry) [inline web/src/App.js]: Remove extra line

File: web/src/components/landing-header.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
+import React from 'react';
```
Comment (shberry) [inline web/src/components/landing-header.js]: Try to list imports in lexicographical order by name

File: web/src/components/landing-header.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+    color:'white',
+    textAlign: 'center',
+    height: 400,
+    justifyContent: 'center',
+    alignItems: 'center'
+  },
+
+}));
+
+// Sub Explanation css style
```
Comment (shberry) [inline web/src/components/landing-header.js]: What is Sub Explanation?

File: web/src/components/landing-header.js [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+    width: '100%',
+    backgroundImage: `url(${landing_bg})`,
+    backgroundSize: 'cover',
+    color:'white',
+    textAlign: 'center',
+    height: 400,
+    justifyContent: 'center',
+    alignItems: 'center'
+  },
+
```
Comment (shberry) [inline web/src/components/landing-header.js]: Remove extra line

File: web/src/components/landing-view-button.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import {withRouter} from 'react-router-dom'
+import {Button,
+        Grid,
+        Box} from '@material-ui/core';
+
+// Popular Text and View All Button redirecting to event list page in Landing Page
```
Comment (shberry) [inline web/src/components/landing-view-button.js]: You can say
Redirect 'Popular' and 'View All' to. ..

File: web/src/components/landing-header.js (hunk 3 above)
Comment (Lhyejin) [reply web/src/components/landing-header.js]: It's sub explanation(== product benefit?) under main header. Is that name not good?

File: web/src/App.js [hunk 6]
```diff
@@ -1,39 +1,39 @@
-import React from 'react';
-import Landing from './pages/landing';
 import ApolloClient from "apollo-boost";
 import { ApolloProvider } from "react-apollo";
 import {
-    BrowserRouter as Router,
-    Switch,
-    Route,
-    Link
+  BrowserRouter as Router,
```
Comment (shberry) [inline web/src/App.js]: Alphabetical order among these too?

File: web/src/components/landing-header.js [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+  return (
+    <div className={subStyle.root}>
+      <Grid container justify="space-between" className={subStyle.text}>
+        <Grid item xs='auto' sm={3}>
+          <img src={benefit[0]} className={subStyle.img} />
+          <Typography variant="h6" gutterBottom>
+            Learn from a guru
+          </Typography>
+          <Typography>
+            Lorem ipsum door sit amet, consectetur adipiscing
```
Comment (shberry) [inline web/src/components/landing-header.js]: Add a TODO to remove this placeholder and add actual text

File: web/src/components/landing-header.js [hunk 8]
```diff
@@ -0,0 +1,113 @@
...
+            elit, sed do elusmo
+          </Typography>
+        </Grid>
+      </Grid>
+    </div>
+  )
+}
+
+// Header in Landing Page including main & sub
+export default function LandingHeader() {
```
Comment (shberry) [inline web/src/components/landing-header.js]: Can we order the functions to match the order in the UI?

File: web/src/components/landing-header.js [hunk 9]
```diff
@@ -0,0 +1,113 @@
...
+            Lorem ipsum door sit amet, consectetur adipiscing
+            elit, sed do elusmo
+          </Typography>
+        </Grid>
+      </Grid>
+    </div>
+  )
+}
+
+// Header in Landing Page including main & sub
```
Comment (shberry) [inline web/src/components/landing-header.js]: Usually, header refers to the top section of this page, which is 'Tiny Social'. This section can be renamed to Body Summary? This is not the best name, so feel free to give this a thought.

File: web/src/components/landing-header.js (hunk 7 above)
Comment (shberry) [reply web/src/components/landing-header.js]: Please see the below comments. One suggestion can be Body Details (or something similar).

File: web/src/components/landing-header.js (hunk 8 above)
Comment (Lhyejin) [reply web/src/components/landing-header.js]: yes, I sorted in order of UI.

File: web/src/components/landing-header.js (hunk 9 above)
Comment (Lhyejin) [reply web/src/components/landing-header.js]: I think that 'Landing Description' is good. so I separated Main Landing Description & Sub Landing Description. what do you think about?

File: web/src/components/landing-description.js [hunk 10]
```diff
@@ -0,0 +1,117 @@
...
+  text: {
+    textAlign: 'center'
+  }
+}));
+
+// Main Description in Landing Page
+function MainLandingDescription() {
+  const mainDescriptionStyles = MainDescriptionStyles();
+  return(
+    // TODO(Lhyejin): Add actual text
```
[the remark this answers is not in the document]
Comment (Lhyejin) [reply web/src/components/landing-description.js]: It's short summary which  will located on each sub landing description section. Now, there's temporary summary for each section as same text.

File: web/src/components/landing-description.js [hunk 11]
```diff
@@ -0,0 +1,117 @@
...
+    <Grid container className={mainDescriptionStyles.root}>
+      <Grid item xs={12}>
+        <Typography variant='h4' paragraph>
+          Reinventing the Book club
+        </Typography>
+        <Typography variant='h6' paragraph>
+          Meaningful Conversations With<br/>
```
[the remark this answers is not in the document]
Comment (Lhyejin) [reply web/src/components/landing-description.js]: if don't add 'br' tag, It disconnects awkwardly in mobile UI.
Also, that sentence was separated in designed Desktop or Mobile screen.

**Pull request merged**